### PR TITLE
Update charter.md

### PR DESCRIPTION
**Pull request by jayunit100**
Updating the sig-windows charter to providing external guidance around the evolving windows ecosystem for orchestration and installation.

### CODE REVIEW

**Review by markyjackson-taulia — CHANGES_REQUESTED** [review]
/lgtm

**Comment by mrbobbytables** [comment]
/lgtm cancel
This needs steering review

File: sig-windows/charter.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Windows specific implementation.
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
+- Providing community guidance for external tools and paradigms necessary for orchestrating the windows stack, including `wins`, `csi-proxy`, and the umbrella (windows privileged containers)[https://github.com/ambguo/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/20201981-windows-privileged-container-support.md] initiative.
```
Comment (dims) [inline sig-windows/charter.md:23]: `ambguo`?

Comment (marosset) [inline sig-windows/charter.md:23]: I'm not sure it is a good idea to call our specific tools here (especially wins where usage is contentious due to its potential for bypassing a lot of security boundaries).

Comment (jayunit100) [reply sig-windows/charter.md]: updated

Comment (jayunit100) [reply sig-windows/charter.md]: i think the link works ?

Comment (jayunit100) [reply sig-windows/charter.md]: (resolve for now)

File: sig-windows/charter.md [hunk 2]
```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
...
 
-- Windows specific code in all parts of the codebase.
+- Windows specific code in all parts of the codebase
 - Testing of Windows specific features and clusters
 
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
-
+- Providing community guidance for external tools and paradigms necessary for production windows k8s administration
```
Comment (neolit123) [inline sig-windows/charter.md:23]: `Windows` should be uppercase like the rest of the doc.

**Comment by dims** [comment]
@michmike Please review!

File: sig-windows/charter.md (hunk 2 above)
Comment (michmike) [reply sig-windows/charter.md:23]: agreed

File: sig-windows/charter.md [hunk 3]
```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
...
+- Helping to transition from CSI/CNI tooling workarounds to the emerging (windows privileged containers)[https://github.com/ambguo/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/20201981-windows-privileged-container-support.md] initiative
```
Comment (michmike) [inline sig-windows/charter.md:24]: i would remove that part right now. privileged containers is unproven yet and we have not made any decisions on long term impact.

**Review by michmike — CHANGES_REQUESTED** [review]
i asked for some changes.

File: sig-windows/charter.md (hunk 3 above)
Comment (jayunit100) [reply sig-windows/charter.md:24]: hmm, ok, i guess part of my goal was to actually get us to actively own the runtime story, maybe thats not in the interests of the sig in general?  thats fine if thats the case - maybe we should be explicit about that?

Comment (jayunit100) [reply sig-windows/charter.md:24]: ^ might be worthy of creating a "Windows container runtime" working group  ....... under the braoder sig ?

Comment (jayunit100) [reply sig-windows/charter.md]: updatedd to focus on reference application rather then wins/privileged containers

**Comment by dims** [comment]
Can we please get sign offs from the chairs and TL's?

@marosset @michmike @benmoss @ddebroy

File: sig-windows/charter.md [hunk 4]
```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
...
-- Windows specific code in all parts of the codebase.
+- Windows specific code in all parts of the codebase
 - Testing of Windows specific features and clusters
 
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
-
+- Providing community guidance for external tools and paradigms necessary for production Windows k8s administration
+- Providing a reference implementation(s) of an end-to-end integrated Windows cluster that can be used for R&D, prototyping, and testing by other SIGs or companies interested in building k8s Windows integrations (this is motivated by the fact that container runtime technologies may change over time and the community benefits from having a canonical example of how to use Windows with K8s).
```
Comment (michmike) [inline sig-windows/charter.md:24]: i would remove that last sentence...IMHO we are not committing to a reference implementation.

**Review by michmike — CHANGES_REQUESTED** [review]

**Comment by mrbobbytables** [comment]
Is this in a state where it should be reviewed by steering?

**Pull request closed without merging**